src/lib/serviceRoutines/postSubscribeContextAvailability.cpp

```diff
@@ -52,7 +52,11 @@ std::string postSubscribeContextAvailability
   SubscribeContextAvailabilityResponse  scar;
   std::string                           answer;  
 
-  TIMED_MONGO(ciP->httpStatusCode = mongoSubscribeContextAvailability(&parseDataP->scar.res, &scar, ciP->uriParam, ciP->tenant));
+  TIMED_MONGO(ciP->httpStatusCode = mongoSubscribeContextAvailability(&parseDataP->scar.res,
```

Comment:
This is the only part of code touched in this PR (to fix a small bug with Fiware-Correlator generation).